website/components/hub/Plugins.tsx

```diff
@@ -0,0 +1,11 @@
+import type { Plugins as PluginsDetails } from "../../content/hub";
+
+export const Plugins = ({ plugins }: { plugins: PluginsDetails }) => {
```

Comment:
Stub component for the plugins overview, supposed to replace https://hub.cloudquery.io/providers